dashboard/src/modules/components/ProfileComponent/index.jsx

```diff
@@ -54,7 +54,7 @@ const ProfileComponent = () => {
                       {
                         <TextContent>
                           <Text component={TextVariants.h5}>
-                            {user?.first_name}
+                            {keycloak.tokenParsed?.given_name}
```

Comment:
> (I think we should be seriously considering removing this whole page....)

Yes, so I wouldn't worry about whether the value is undefined or not at this point. We will eventually remove it _I think_

Reply:
Yeah, but the problem is that right now we know that there is an issue here.  If we ignore it now because we might remove the page later, then when later comes we might not remember that there was an issue here.

At the moment, we have no need (that I am aware of) for this page.  So, why not remove it?  That gets rid of this issue, as well as any further concerns about maintaining this code.  If in the future we have a need for a page like this one, we can reinvent it from a clean slate and not have to consider what the implications of changing existing code might be; and, in the meantime, we can make changes elsewhere without having to consider their impacts on the code here.

Reply:
I have created a discussion topic https://github.com/distributed-system-analysis/pbench/discussions/3360. Lets discuss over there what to do with profile page and user creation date.